playbooks/roles/common_vars/defaults/main.yml

```diff
@@ -14,13 +14,17 @@ COMMON_BASIC_AUTH_EXCEPTIONS:
 # Settings to use for calls to edxapp manage.py
 COMMON_EDXAPP_SETTINGS: 'aws'
 
+# Set to True to install aws role when running edx_sandbox playbook.
+COMMON_ENABLE_AWS_INTEGRATION: False
+
 # Turn on syncing logs on rotation for edx
 # application and tracking logs, must also
 # have the aws or openstack role installed
 COMMON_OBJECT_STORE_LOG_SYNC: False
```

Comment:
@Metfriet, have to check if this is set to `True` in the `configuration-secure` repository.